vaxrank/mutant_protein_fragment.py

```diff
@@ -166,3 +168,23 @@ def top_k_subsequences(
         subsequences = list(self.generate_subsequences(subsequence_length))
         subsequences.sort(key=sort_key)
         return subsequences[:k]
+
+    def predicted_effect(self):
+        effects = [self.variant.effect_on_transcript(t) for t in
+            self.supporting_reference_transcripts]
+        predicted_effect = top_priority_effect(effects)
+        return predicted_effect
+
+    def global_start_pos(self):
+        # position of mutation start relative to the full amino acid sequence
+        global_mutation_start_pos = self.predicted_effect().aa_mutation_start_offset
```

Comment:
I think that Vaxrank has some unresolved tension around how to interpret Isovar sequences which may differ from the reference at more positions that those predicted from DNA by Varcode. If there's an adjacent somatic variant changing an additional AA -- do we count that as part of the mutation? I guess we can't because that difference could also be germline. 

I think in the long term we need Isovar to take a second VCF (of germline variants) and use that to attribute AA differences as either germline or tumor. 

In the mean time, I guess using Varcode's predicted protein pos is OK.